### ISSUE TITLE
An Option to export the Receipt in txt format

### ISSUE THREAD

**Issue by ernest-ns** [aniket-k-kaushik/sales-tax-calculator#12]
I was not able to see an option to export the receipt in txt format as discussed in the email thread. 
I think Jerry mentioned that only the PDF export was optional.

Can you add an option to export to text?

**Comment by jerryjohnjacob** [comment]
That's right. The txt format is required, but there is no need to use any non-native library to generate the file. Do keep presentation in mind though.

**Comment by aniketkaushik** [comment]
I can make make pdf download
